packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md

```diff
@@ -5,42 +5,61 @@ title: Upgrade your Okta Sign-In Widget
 <ApiLifecycle access="ie" /><br>
 <ApiLifecycle access="Limited GA" />
 
-## Overview
+
+This guide covers how to upgrade the Okta Sign-In Widget, which depends on whether your user authentication deployment model is a redirect authentication or an embedded authentication.
+
+---
+**Learning outcomes**
+
+* Understand the benefits of keeping your Sign-In Widget up to date.
```

Comment:
should "up to date" be written "up-to-date"? Usually is, in my experience, but I wonder if this is a style guide thing.

Reply:
@chrisdmills-okta it is hyphenated if it precedes a noun. It's not hyphenated if it comes after a noun.
Example:
You can keep an up-to-date Sign-In Widget.
You can keep your Sign-In Widget up to date.

Reply:
Ah, that makes sense — thanks for clarifying!